akka-docs/rst/scala/http/introduction.rst

```diff
@@ -25,9 +25,8 @@ Akka HTTP was designed specifically as “not-a-framework”, not because we don
 
 Using Akka HTTP
 ---------------
-Akka HTTP is provided in separate jar files, to use it make sure to include the following dependencies::
+Akka HTTP is provided in a separate jar file, to use it make sure to include the following dependency::
 
-  "com.typesafe.akka" %% "akka-http-core" % "@version@" @crossString@
   "com.typesafe.akka" %% "akka-http-experimental" % "@version@" @crossString@
```

Comment:
Not quite sure about this change... We do want to explain that it's 2 modules.

How about explaining here that: hey, you should likely use `"com.typesafe.akka" %% "akka-http-experimental" % "@version@" @crossString@` however you should know that the core is actually in a separate module?